Formula/apache-httpd.rb

```diff
@@ -0,0 +1,114 @@
+class ApacheHttpd < Formula
+  desc "Apache HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
+    sha256 "d35e67ae745053cda23273a520ead6bde506ec8082057edcd3185c9e36eae483" => :sierra
+    sha256 "4ea7056d6c08f9cf49507869aa8c78e3522c601d87b3347d31561ba05b33d41c" => :el_capitan
+    sha256 "e6ae5c4d38b40bcf3d85877ac117a1ea048de774edec3c2b0ff714bc01a6d40a" => :yosemite
+  end
+
+  deprecated_option "with-http2" => "with-nghttp2"
+
+  depends_on "apr"
+  depends_on "apr-util"
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "nghttp2"
+
+  def install
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    args = %W[
+      --enable-mods-shared=all
+      --enable-pie
+      --with-apr=#{Formula["apr"].opt_prefix}
+      --with-apr-util=#{Formula["apr-util"].opt_prefix}
+      --with-pcre=#{Formula["pcre"].opt_prefix}
+      --with-ssl=#{Formula["openssl"].opt_prefix}
+      --with-nghttp2=#{Formula["nghttp2"].opt_prefix}
+      --with-port=8080
+      --with-sslport=8443
+      --enable-mpms-shared=all
```

Comment:
Should we also include `--with-mpm=prefork` so the default config file gets written with that module loaded? If not, the event mpm will be written to the default config that people start with (not great for people that have come to expect the default to be prefork).

Reply:
I think we should default to the upstream default.